site/src/pages/OrganizationSettingsPage/IdpSyncPage/IdpSyncPage.tsx

```diff
@@ -40,13 +44,34 @@ export const IdpSyncPage: FC = () => {
 		rolesQuery,
 	] = useQueries({
 		queries: [
-			groupIdpSyncSettings(organizationName),
-			roleIdpSyncSettings(organizationName),
+			{
+				...groupIdpSyncSettings(organizationName),
+				onSuccess: (data: GroupSyncSettings) => {
+					if (data?.field) {
+						setGroupClaimField(data.field);
+					}
+				},
```

Comment:
🪓 `onSuccess`

for reference: https://tkdodo.eu/blog/breaking-react-querys-api-on-purpose